### DISCUSSION TITLE
AI Builder, Flow apps deployment through build tools

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/powerplatform-build-tools#1014] [Q&A]
Can we use power platform build tools to manage ALM for AI builder and Flow apps?  Is there any guidance available?  Much appreciated.

**Comment by jbuenoliveira** [comment]
@ali-khawaja  - I believe that if your components are in a solution, you can use the `pac solution export` and `import` commands to move them across environments.
https://learn.microsoft.com/en-us/ai-builder/byom-alm
